Ginger/GingerCoreNET/AnalyzerLib/AnalyzeAction.cs

```diff
@@ -240,55 +240,63 @@ public static List<AnalyzerItemBase> Analyze(BusinessFlow BusinessFlow, Activity
                 {
                     if (ARV.StoreTo != ActReturnValue.eStoreTo.None)
                     {
-                        string StoreToValue = ARV.StoreToValue;
-                        ActReturnValue.eStoreTo StoreToType = ARV.StoreTo;
-                        if (StoreToType == ActReturnValue.eStoreTo.Variable)
+                        bool issueFound = false;
+                        if (string.IsNullOrEmpty(ARV.StoreToValue))
                         {
-                            if (BusinessFlow.GetAllVariables(parentActivity).Where(x => x.Name == StoreToValue).Select(x => x.Name).FirstOrDefault() == null)
-                            {
-                                AnalyzeAction AA = CreateNewIssue(BusinessFlow, parentActivity, a);
-                                AA.Description = "Output Values is missing " + GingerDicser.GetTermResValue(eTermResKey.Variable) + " '" + StoreToValue + "'";
-                                AA.Details = "'" + StoreToValue + "' does not exist In " + GingerDicser.GetTermResValue(eTermResKey.BusinessFlow) + " '" + BusinessFlow.Name + "' => " + GingerDicser.GetTermResValue(eTermResKey.Activity) + " '" + parentActivity.ActivityName + "' =>" + "Action '" + a.Description + "' ";
-                                AA.HowToFix = "Remap Output Values";
-                                AA.CanAutoFix = AnalyzerItemBase.eCanFix.No;
-                                AA.IssueType = eType.Error;
-                                AA.Impact = "Execution will fail in run time";
-                                AA.Severity = eSeverity.Critical;
-
-                                IssuesList.Add(AA);
-
-                            }
+                            issueFound = true;
                         }
-                        else if (StoreToType == ActReturnValue.eStoreTo.DataSource)
+                        else
                         {
-                            if (StoreToValue != "" && StoreToValue != null)
+                            switch (ARV.StoreTo)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)